lightning/src/ln/onchain_utils.rs

```diff
@@ -0,0 +1,90 @@
+//! Utilities for computing witnesses weight and feerate computation for onchain operation
+
+use ln::msgs::DecodeError;
+use util::ser::{Readable, Writer, Writeable};
+
+#[derive(PartialEq, Clone, Copy)]
+pub(crate) enum InputDescriptors {
```

Comment:
Could be left for a later clean up: s/InputDescriptors/InputDescriptor

Reply:
InputDescriptors are removed after this PR, so not sure if it's worthy to correct.